references/cli/addon.go

```diff
@@ -131,12 +131,14 @@ func NewAddonEnableCommand(c common.Args, ioStream cmdutil.IOStreams) *cobra.Com
 		Use:   "enable",
 		Short: "enable an addon",
 		Long:  "enable an addon in cluster.",
-		Example: `Enable addon by:
+		Example: `  Enable addon by:
 	vela addon enable <addon-name>
-Enable addon with specify version:
+  Enable addon with specify version:
 	vela addon enable <addon-name> --version <addon-version>
-Enable addon for specific clusters, (local means control plane):
+  Enable addon for specific clusters, (local means control plane):
 	vela addon enable <addon-name> --clusters={local,cluster1,cluster2}
+  Enable addon locally:
+	vela addon enable <your-local-addon-path>
```

Comment:
Also add an example about how to specify parameters?

Reply:
yes, good idea